### PR TITLE
Fsa22 v2 240 component details page banner

### PR DESCRIPTION
**Pull request by tisidro**
### Description:

Component: Details Page Banner

### Spec:

Designs: [Designs](DESIGN_URL)

See Story: [FSA22V2-240](https://sparkbox.atlassian.net/browse/FSA22V2-240?atlOrigin=eyJpIjoiOGJiNjVhYmY4MDM3NDY2NmJiYzRhY2Q1MjhhZDQ0OWIiLCJwIjoiaiJ9)

### Validation:

So far this is what is done:
1. Component is built
2. Component is linked to id.page.jsx
3. Component has working test

Working on: 
1. Formatting to design spec.
2. Importing images shown on Figma design.

- [ ] This PR has code changes, and our linters still pass.
- [ ] This PR affects production code, so it was browser tested (see below).
- [ ] This PR has new code, so new tests were added or updated, and they pass.

#### To Validate:

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
4. Pull down all related branches.
5. In terminal: `npm run dev`.
6. Check [localhost:3000](http://localhost:3000/)for changes.
7. Run `npm run lint`.
8. Run axe Devtools on affected pages.
9. Confirm all tests pass.

- There is a DetailsBanner component that accepts the props of name: string (used in an h1 tag) and an image object.
- The component is linked in the [id].pages.jsx page.
- There is a test that checks for dynamic content in the h1 and the image.
- The layout is responsive and matches [this figma design](https://www.figma.com/file/UjMQEwUvAsnFAMhYNdUiVT/Apprenticeship-Capstone?node-id=244%3A617).

---

#### Browser Testing

In these browsers, behavior & design closely match original specifications. A user is able to access all content and functionality, including the usability of required assistive devices, such as keyboard and screenreader.

**macOS**

- [ ] Safari (last 2 major versions)
- [ ] Chrome (last 6 months)
- [ ] Firefox (last 6 months)

**Windows**

- [ ] Chrome (last 6 months)
- [ ] Firefox (last 6 months)
- [ ] Edge (last 6 months)

**Mobile**

- [ ] Safari (last 2 major versions)
- [ ] Chrome on Android (last 6 months)
- [ ] Firefox on Android (last 6 months)

### CODE REVIEW

**Review by empee3 — APPROVED** [review]

File: src/components/DetailsBanner.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export default function DetailsBanner({ name, image }) {
+  const { src, alt } = image; // destructure from image object
+  return (
+    <div className="details-banner">
+      {name}
```
Comment (jkohrt7) [inline src/components/DetailsBanner.jsx:5]: Axe Devtools is asking for a level one header on the page. I'd wrap the {name} here in an h1 tag to see if that solves the issue.

**Review by jkohrt7 — DISMISSED** [review]
Before you squash, add that h1 header and check that the associated axe devtools error goes away in any/all of the [id] pages.

**Review by jkohrt7 — DISMISSED** [review]
All the checks pass--great work!

**Review by jkohrt7 — APPROVED** [review]
Ok, the squashed commit looks good.

**Pull request merged**